Homework_2.playground/Contents.swift

```diff
@@ -0,0 +1,96 @@
+
+class Stack {
+    private var head: Frame? = nil
+    
+    func push(element: Int) {
+        if let notNilHead = head {
```

Comment:
```suggestion
        if let head = head {
```

typically you don't need to add prefix like `notNil`